src/BlogContent/AQUA-Evaluation/aqua_execute_query.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+####################################################################################################################
+#
+# This script execute  queries  from capture.sql file generated earlier with   AQUA turned On and Off
+# Arguments are: 
+#           -h PGHOST
+#           -p PGPORT
+#           -U PGUSR
+#           -d PGDATABASE
+#           -t PGCONNECT_TIMEOUT
+# sample run with parameters below :
+#./aqua_execute_query.sh -h 3.88.94.167 -p 5439 -d dev -U test_user 
+#######################################################################################################################
+
+
+
+
+while getopts h:p:U:d:t:  option; do
+      case "${option}"
+      in
+      h) export PGHOST=${OPTARG};;
+      p) export PGPORT=${OPTARG};;
+      d) export PGDATABASE=${OPTARG};;
+      U) export PGUSER=${OPTARG};;
+      t) export PGCONNECT_TIMEOUT=${OPTARG};;
+      esac
+done
+
+read -s -p "Password to connect to Redshift cluster: " PGPASSWORD
+export  PGPASSWORD=$PGPASSWORD
+echo
+if [[ -z $PGCONNECT_TIMEOUT ]]; then
+  export PGCONNECT_TIMEOUT=10
+fi
+
+#TODO: Validate ALL inputs as shown below
+
+if [[ -z $PGHOST ]] || [[ -z $PGPORT ]] || [[ -z $PGDATABASE ]] || [[ -z $PGUSER ]] ; then
+echo "connection parameters required and cannot be empty, please enter correct  hostname, dbname, port-number, username to connect"     
+   exit -1
+fi
+
+if [ ! -s ./capture.sql  ]; then
+
+echo "The capture.sql seems empty, please capture aqua eligible queries using aqua_capture_query.sh then execute aqua_execute_query.sh"
+else
+#log execution startime 
```

Comment:
Is there a reason there is no indentation for these scripts?  Can you properly indent for readability?